lib/metadata/bucketfile/backend.js

```diff
@@ -0,0 +1,402 @@
+import { errors } from 'arsenal';
+import level from 'level';
```

Comment:
I noticed that `level` is not part of package.json, it needs to be added to the list of dependencies.